src/main/resources/frontend/src/views/AdminView/SubmissionsView.vue

```diff
@@ -68,10 +85,41 @@ const columnDefs = reactive([
 const rowData = reactive({
   value: []
 })
+
+const adminSubmit = async (phase: Phase) => {
+  if (adminRepo == "") {
+    alert("Please enter a repo url")
+    return
+  }
+  try {
+    await adminSubmissionPost(phase, adminRepo)
+    runningAdminRepo.value = true;
+  } catch (error) {
+    if (error instanceof Error) { alert("Error running grader: " + error.message) }
+    else { alert("Unknown error running grader") }
+  }
+}
 </script>
 
 
 <template>
+
+  <div class="adminSubmission">
+    <input v-model="adminRepo" type="text" id="repoUrlInput" placeholder="Github Repo URL"/>
+    <Dropdown>
+      <template v-slot:dropdown-parent>
+        <button>Grade Repo</button>
+      </template>
+      <template v-slot:dropdown-items>
+        <a @click="adminSubmit('0')">Phase 0</a>
+        <a @click="adminSubmit('1')">Phase 1</a>
+        <a @click="adminSubmit('3')">Phase 3</a>
+        <a @click="adminSubmit('4')">Phase 4</a>
+        <a @click="adminSubmit('5')">Phase 5</a>
+      </template>
```

Comment:
What would you think about using something like `<template v-slot:dropdown-items :disabled="adminRepo.length > 0">`? With that we could avoid error handling on lines 90-93 and make the requirement more intuitive

Reply:
I think this is a brilliant idea! It took a minute to figure it out, but now the dropdown is completely disabled, unless the textbox contains at least `github.com/`